docs/tutorials/slack-history.md

```diff
@@ -0,0 +1,90 @@
+# Saving Slack Messages
+
+## Overview
+
+The [slack free-tier](https://slack.com/pricing/paid-vs-free) saves only the last 10K messages. For social Slack instances, it may be impractical to upgrade to a paid plan to retain these message. Similarly, for an OSS project like Airbyte where we interact with our community through a public slack instance, the cost of paying for a seat for every slack member is inhibitive.
+
+Searching through old messages can be really helpful. For me, losing that history feels like some advanced form of memory loss. What was that funny thing that someone said about chicken tenders? This contributor question sounds familiar, haven't we seen it before? But you just can't remember!
+
+This tutorial will show you how you can, for _free_, use Airbyte to save these message (even after Slack deletes them). It will also provide you a convenient way to search through them.
+
+Specifically we will export messages from your slack instance into search engine called MeiliSearch. We will be focusing on getting this setup running from your local workstation. We will mention at the end how you can set up a more productionized version of this pipeline.
+
+We want to make this process easy, so while we will link to some external documentation for further exploration, we will provide all the instructions you need here to get this up and running.
+
+## Set up Airbyte
+
+Make sure you have Docker and Docker Compose installed. Then run the following commands:
+
+```text
+git clone https://github.com/airbytehq/airbyte.git
+cd airbyte
+docker-compose up
+```
+
+If you run into any problems, feel free to check out our more extensive [getting started](../getting-started.md) for more help.
+
+Once you see an Airbyte banner, the UI is ready to go at [http://localhost:8000/](http://localhost:8000/). You can set your user preferences and then you will be brought to a page that asks you to set up a source. In the next step we'll go over how to do that.
+
+
+## Connect to Slack
+
+In the Airbyte UI select Slack from the dropdown. We provide step-by-step instructions for setting up the Slack source in Airbyte [here](../integrations/sources/slack.md). These will walk you through how to complete the form on this page.
+
+By the end of these instructions you should have created a Slack source in the Airbyte UI. The Airbyte app will now prompt you to set up a destination. Next, we will now walk through how to set up MeiliSearch.
+
+## Set up MeiliSearch
+
+First let's get MeiliSearch running on our workstation. MeiliSearch has extensive docs for [getting started](https://docs.meilisearch.com/learn/tutorials/getting_started.html#download-and-launch). For this tutorial, however, we will give you all the instructions you need to set up MeiliSearch using docker.
```

Comment:
You could fin the documentation for docker on [the reference] (https://docs.meilisearch.com/reference/features/installation.html#download-and-launch)